### PR TITLE
request: development <- 57 fix minor issues annotations

### PR DESCRIPTION
**Pull request by Cusatelli**

### CODE REVIEW

**Comment by Cusatelli** [comment]
# ⚠️ Please do not merge before everyone has reviewed this pull-request.

File: src/main/java/com/noroff/mefit/security/SecurityConfig.java [hunk 1]
```diff
@@ -32,21 +27,19 @@ protected void configure(HttpSecurity http) throws Exception {
...
-                            .antMatchers("/user/**").hasAnyRole("MeFitt_User")
-                            .antMatchers("/security/admin").hasAnyRole("MeFitt_Admin")
-
-                            // All remaining paths require authentication
-                            .anyRequest().authenticated();
-                })
+                .authorizeRequests(authorize -> authorize
+                        // Specify paths where public access is allowed
+                        .antMatchers("/v3/api-docs", "/v3/api-docs/*", "/v3/api-docs/**").permitAll()
+                        .antMatchers("/swagger-ui", "/swagger-ui/*", "/swagger-ui/**").permitAll()
```
Comment (Cusatelli) [inline src/main/java/com/noroff/mefit/security/SecurityConfig.java:33]: Database or KeyCloak public access points?

File: src/main/java/com/noroff/mefit/security/SecurityConfig.java [hunk 2]
```diff
@@ -32,21 +27,19 @@ protected void configure(HttpSecurity http) throws Exception {
...
+
+                        /*
+                         * Add urls here
+                         */
+                        .antMatchers("/user/**").hasAnyRole("MeFit_User")
+                        .antMatchers("/security/admin").hasAnyRole("MeFit_Admin")
```
Comment (Cusatelli) [inline src/main/java/com/noroff/mefit/security/SecurityConfig.java:39]: Do we need to add all users here manually?

**Review by Cusatelli — COMMENTED** [review]
I can not find anything at this time, though that might be because I am biased.

**Review by OmarAbdiAli — CHANGES_REQUESTED** [review]
schema.sql has a drop if exists thats not working as expected.

**Review by OmarAbdiAli — APPROVED** [review]
Great, its Done!

**Pull request merged**